### PR TITLE
Fix problem with scrambled cards in SV view

### PR DESCRIPTION
**Pull request by moonso**
**Prepare**
1. - [x] look at a SV page with local observations active, some cards are scrambled  

**How to test**:
1. - [x] Install on stage or local
1. - [x] Repeat above and check that it looks fine

**Review:**
- [x] code approved by
- [x] tests executed by

### CODE REVIEW

File: scout/server/blueprints/cases/controllers.py [hunk 1]
```diff
@@ -173,13 +160,24 @@ def case(store, institute_obj, case_obj):
...
-        )
+        panel_name = panel_info["panel_name"]
+        panel_version = panel_info.get("version")
+        panel_obj = store.gene_panel(panel_name, version=panel_version)
+        if not panel_obj:
+            LOG.warning(
+                "Could not fetch gene panel %s, version %s", panel_name, panel_version
+            )
+            LOG.info("Try to fetch latest existing version")
+            panel_obj = store.gene_panel(panel_name)
```
Comment (dnil) [inline scout/server/blueprints/cases/controllers.py]: Sure! I suppose one could also imagine the store having that, but that is just style. This is a new functionality, while not huge may be worth a little note somewhere?

File: scout/server/blueprints/variant/templates/variant/sv-variant.html [hunk 2]
```diff
@@ -169,46 +81,45 @@
...
   </div>
 
   <div class="row">
     <div class="col-12">
       <div class="card">
-        <nav>
-          <div class="nav nav-tabs" id="nav-tab" role="tablist">
-            <a class="nav-item nav-link active" id="nav-genes-tab" data-toggle="tab" href="#nav-genes" role="tab" aria-controls="nav-genes" aria-selected="true">Genes</a>
-            <a class="nav-item nav-link" id="nav-transcripts-tab" data-toggle="tab" href="#nav-transcripts" role="tab" aria-controls="nav-transcripts" aria-selected="false">Transcripts</a>
+        <div class="panel-heading">
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/sv-variant.html]: 👍🏻

File: scout/server/blueprints/variant/templates/variant/sv-variant.html [hunk 3]
```diff
@@ -220,166 +131,103 @@
     {# Email setting must be setup #}
     {{ modal_cancel_verify() }}
   {% endif %}
+</div>
 {{ modal_causative(case, institute, variant) }}
 {% endblock %}
 
 {% macro panel_basics() %}
-  <div class="card mt-3 panel-default">
+  <div class="card panel-default">
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/sv-variant.html]: Eh there was an unused panel_basics for svs? Ok.. 😊

File: scout/server/blueprints/variant/templates/variant/variant_details.html [hunk 4]
```diff
@@ -118,17 +118,17 @@
...
+        {% if observations['cases'] %}
+          {% for data in observations['cases'] %}
+            {% if data.variant %}
+              <a class="ml-3" href="{{ url_for('variant.variant', institute_id=data.case.owner, case_name=data.case.display_name, variant_id=data.variant._id) }}">{{ data.case.display_name }}</a>
+            {% else %}
+              <span class="text-muted ml-3">{{ data.case.display_name }}</span>
+            {% endif %}
+          {% endfor %}
+        {% endif %}
+    </div>
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/variant_details.html]: 😝Well found!

**Comment by moonso** [comment]
Ok @dnil I'm done

**Comment by dnil** [comment]
coveralls approves! :wink: Thank you!

**Pull request merged**